src/amo/components/SearchForm/index.js

```diff
@@ -49,24 +50,34 @@ export class SearchFormBase extends React.Component<Props> {
   }
 
   render() {
-    const { className, i18n } = this.props;
+    const { className, i18n, apiLang, clientApp } = this.props;
 
     return (
-      <form
-        action={this.baseSearchURL()}
-        className={makeClassName('SearchForm', className)}
-        method="GET"
-        data-no-csrf
-        role="search"
-      >
-        <AutoSearchInput
-          inputName="q"
-          onSearch={this.onSearch}
-          onSuggestionSelected={this.onSuggestionSelected}
-          selectSuggestionText={i18n.gettext('Go to the add-on page')}
-          showInputLabel={false}
-        />
-      </form>
+      <React.Fragment>
+        <Helmet>
+          <link
+            title={`Firefox Add-ons (${apiLang || ''})`}
+            rel="search"
+            type="application/opensearchdescription+xml"
+            href={`/${apiLang || ''}/${clientApp || ''}/opensearch.xml`}
```

Comment:
We should always have a `apiLang` and `clientApp` so I would remove the `|| ''`.

Reply:
I see. Let's keep it as is then. I see that's done in another method of this component anyway.